app/src/main/java/com/example/android/sunshine/app/DetailFragment.java

```diff
@@ -103,44 +107,45 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         Bundle arguments = getArguments();
         if (arguments != null) {
-            mUri = arguments.getParcelable(DetailFragment.DETAIL_URI);
+            uri = arguments.getParcelable(DETAIL_URI);
+            transitionAnimation = arguments.getBoolean(DETAIL_TRANSITION_ANIMATION);
         }
 
-        View rootView = inflater.inflate(R.layout.fragment_detail, container, false);
-        mIconView = (ImageView) rootView.findViewById(R.id.detail_icon);
-        mDateView = (TextView) rootView.findViewById(R.id.detail_date_textview);
-        mFriendlyDateView = (TextView) rootView.findViewById(R.id.detail_day_textview);
-        mDescriptionView = (TextView) rootView.findViewById(R.id.detail_forecast_textview);
-        mHighTempView = (TextView) rootView.findViewById(R.id.detail_high_textview);
-        mLowTempView = (TextView) rootView.findViewById(R.id.detail_low_textview);
-        mHumidityView = (TextView) rootView.findViewById(R.id.detail_humidity_textview);
-        mWindView = (TextView) rootView.findViewById(R.id.detail_wind_textview);
-        mPressureView = (TextView) rootView.findViewById(R.id.detail_pressure_textview);
+        View rootView = inflater.inflate(R.layout.fragment_detail_start, container, false);
+        iconView = (ImageView) rootView.findViewById(R.id.detail_icon);
+        dateView = (TextView) rootView.findViewById(R.id.detail_date_textview);
+        descriptionView = (TextView) rootView.findViewById(R.id.detail_forecast_textview);
+        highTempView = (TextView) rootView.findViewById(R.id.detail_high_textview);
+        lowTempView = (TextView) rootView.findViewById(R.id.detail_low_textview);
+        humidityView = (TextView) rootView.findViewById(R.id.detail_humidity_textview);
+        humidityLabelView = (TextView) rootView.findViewById(R.id.detail_humidity_label_textview);
+        windView = (TextView) rootView.findViewById(R.id.detail_wind_textview);
+        windLabelView = (TextView) rootView.findViewById(R.id.detail_wind_label_textview);
+        pressureView = (TextView) rootView.findViewById(R.id.detail_pressure_textview);
+        pressureLabelView = (TextView) rootView.findViewById(R.id.detail_pressure_label_textview);
```

Comment:
Not really, this project is the continuation of an existing one, from the beginner Android course on Udacity, and I generally try to avoid adding new libraries to it. Also, these were already here, I only got rid of the hungarian notation xD